tests/jeos/firstrun.pm

```diff
@@ -67,7 +67,15 @@ sub verify_mounts {
 
 sub run {
     my ($self) = @_;
-
+    # kiwi-templates-JeOS images (sle, opensuse x86_64 only) are build w/o translations
+    # jeos-firstboot >= 0.0+git20200827.e920a15 locale warning dialog has been removed
+    if (is_sle('<15-sp3') || (is_opensuse && is_x86_64)) {
+        assert_screen 'jeos-lang-notice', 300;
+    } elsif (is_opensuse && !is_x86_64) {
+        assert_screen 'jeos-locale', 300;
+    } else {
+        assert_screen 'jeos-keylayout', 300;
+    }
     mouse_hide;    # JeOS on generalhw
```

Comment:
This needs to happen before any `assert_screen`

Reply:
Oops, ofc. thanks!